roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -181,6 +205,29 @@ public Roaring64NavigableMap(
     resetPerfHelpers();
   }
 
+  /**
+   * set the serialization mode.
+   * if the serializationMode is successfully set, the static variable 'SERIALIZATION_MODE' will be ineffective.
+   * if the serializationMode is successfully set, it will impact only FUTURE bitmaps, not the ones already created.
```

Comment:
`   * if the serializationMode is successfully set, it will impact only FUTURE bitmaps, not the ones already created` looks like a comment about the static variable, not the non-static setter.